modules/fastqc.nf

```diff
@@ -9,13 +9,10 @@ process fastqc {
     tuple val(name), path(reads) 
 
     output:
-    tuple val(name), path("*.tar.gz"), emit: tar
+    path("*_fastqc.zip", emit: zip)
 
     script:
     """
-    mkdir -p fastqc_\$(basename \$PWD)
-    fastqc -t ${task.cpus} ${reads} -o fastqc_\$(basename \$PWD)
-    tar zcvf fastqc_\$(basename \$PWD).tar.gz fastqc_\$(basename \$PWD)
-    rm -rf fastqc_\$(basename \$PWD)
+    fastqc --noextract -t ${task.cpus} ${reads}
```

Comment:
prefix of the zip output is the read file name, right? Then this should work

Reply:
yes, it's like `SRR2932639_fastqc.zip` or `mock_treated_fastqc.zip`